portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,22 +24,60 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/** Class used to store comment data */
+class Comment {
+  String name;
+  String commentText;
+  
+  /**
+   * Constructor
+   * @param name Author of comment
+   * @param commentText Comment written by author
+   */
+  public Comment(String name, String commentText) {
+    this.name = name;
+    this.commentText = commentText;
+  }
+}
+
+
 /** Servlet that writes an list of messages as a response. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  /** Hard coded messages used for tesing JSON */
-  private List<String> messages;
+  /** Stores comments added */
+  private List<Comment> comments;
 
   @Override
   public void init(){
-    messages = new ArrayList<>(Arrays.asList("one", "two", "three"));
+    comments = new ArrayList<>();
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String name = getParameter(request, "name-input", "");
+    String commentText = getParameter(request, "comment-input", "");
+
+    comments.add(new Comment(name, commentText));
+    response.sendRedirect("/html/comments.html");
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter
```

Comment:
You want to start the JavaDoc with a summary of what the method does. If the method is basic and the summary covers what is returned, then you do not need to add a @return tag.

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
See above